core/domain/skill_services.py

```diff
@@ -867,3 +867,31 @@ def skill_has_associated_questions(skill_id):
         question_models.QuestionSkillLinkModel.get_all_question_ids_linked_to_skill_id( # pylint: disable=line-too-long
             skill_id))
     return len(question_ids) > 0
+
+
+def filter_skills_by_mastery(user_id, skill_ids):
+    """Given a list of skill_ids, it returns a list of
+    20 (feconf.MAX_NUMBER_OF_SKILL_IDS) skill_ids in which the user has
```

Comment:
Don't mention 20 here as if we change the constant, we would have to change here as well.

Reply:
Done.